src/js/apps/patients/worklist/filters_app.js

```diff
@@ -19,9 +27,32 @@ export default App.extend({
     this.showFilters();
   },
   showFilters() {
+    this.showAllFiltersButtonView();
     this.showGroupsFilterView();
     this.showOwnerToggleView();
   },
+  showAllFiltersButtonView() {
+    if (this.groups.length < 2) return;
+
+    const ownerView = this.showChildView('allFilters', new AllFiltersButtonView());
+
+    this.listenTo(ownerView, 'click', () => {
+      const state = this.getState();
```

Comment:
I think maybe this handler should be `showGroupSidebar`

Reply:
Added a `showGroupSidebar()` handler function in my last push.